CONTRIBUTING

```diff
@@ -0,0 +1,91 @@
+# Contributing
+
+Every contribution is highly appreciated.
+If you have ideas or improvements feel free to create a fork and open a pull requests.
+Issues and engagement in open discussions are also welcome.
+
+## Developers
+
+### Languages
+* We use python3 for rapid prototyping and fast development cycles
+* We started with rust for performance improvements
+
+### Tests
+* We write unit tests for our code.
+* We use pytest for tests.
+* We use pylint to check test coverage. We aim for a coverage of 100%. However, even 100% coverage does not guarantee that everything was fully tested.
+* Every fixed bug should have its own unittest.
+* We write unittests in rust for rust code
+* We write integration tests for python for every exposed interface from rust
+
+### Git Workflow
+* We have a main branch. We do not push to the main branch, please develop on a separate branch and open a pull request to main.
+* We create pull requests in GitHub to merge a branch into main. A maintainer must accept each request.
+* Branches should be named after the feature/fix that they implement. Words should be separated by hyphens. Feature branches should begin with 'dev-' and fix branches with 'fix-' (e.g. 'dev-kafka-connector') or you go with the auto generated branch names from github.
+* The main branch must always be in a functioning state, i.e., all tests pass successfully.
+* We merge  pull requests with "Squash and merge".
+* New commits must pass the unit tests.
+* Rebasing feature branches, e.g., to clean up the git history, is strongly advised before request a review.
+
+### Commit Messages
+* We divide subject and content with a blank line.
+* The subject should not be longer than 50 characters, but 72 is the maximum limit.
+* The subject should begin with a capital letter.
+* The subject should not end with a period.
+* We write commit messages in English.
+* The subject should complete the sentence "If applied, this commit will ..." (e.g., "If applied, this commit will ‘Add some functionality‘").
```

Comment:
Is it really necessary to start every commit with "If applied, this commit will "?
This uses up about half of the available space for the subject and that information is already implied, since it's a commit.
Wouldn't "Add ..." be enough?

Reply:
no, you misunderstood. the commit message to be written should only complete this sentence. It is the same scheme we already use.